### DISCUSSION TITLE
Not finding searched term in subdirectories

### DISCUSSION THREAD

**Discussion by AtomicNess123** [BurntSushi/ripgrep#1601] [General] [answered]
I can seem to find a searched term that is a subdirectory level 3.
When I point to the directory itself or on the level right above, ripgrep finds it.
However, when I try to search it from upper levels, it won't, even if I specify `--max-depth 10`.
I wonder what I am doing wrong.
Thanks!

**Comment by BurntSushi** [comment, marked as answer]
Sorry, but there isn't really any way to answer this without an actual test case that I can reproduce. My guess is that this is related to your gitignores. You can test it by disabling gitignores (`-u`) and seeing if that helps. You can also use `--debug` to see why ripgrep is ignoring certain files or directories.

**Reply by AtomicNess123** [reply]
I actually noticed that the term had an accent (ó), and I was using the word without the accent. Is there a way to find all words independent of the vowel variation (=o, ö, ó)? Thanks.

**Reply by BurntSushi** [reply]
This is a totally different question though. If the same term matched what you wanted using a different directory as input, then this shouldn't be a factor.

> Is there a way to find all words independent of the vowel variation

No. Not unless you know your corpus is in composed normal form. Then you could search for just `o` or even `o\p{Mark}*` I think, and that might do it.

**Reply by AtomicNess123** [reply]
Yes, it's true, my bad. Actually, I opened a new question because the reason was that it was in a DOC document. Thanks for the composed form tip!
